File: killrvideo-service-users/src/main/java/com/killrvideo/service/user/dao/UserMapper.java
package com.killrvideo.service.user.dao;

import com.datastax.oss.driver.api.mapper.annotations.DaoFactory;
import com.datastax.oss.driver.api.mapper.annotations.Mapper;

@Mapper
public interface UserMapper {
    @DaoFactory
    UserDao getUserDao();
}

File: killrvideo-service-users/src/main/kotlin/com/killrvideo/service/user/dao/UserMapper.kt
package com.killrvideo.service.user.dao

import com.datastax.oss.driver.api.mapper.annotations.DaoFactory
import com.datastax.oss.driver.api.mapper.annotations.Mapper

@Mapper
interface UserMapper {
    @get:DaoFactory
    val userDao: UserDao

    @get:DaoFactory
    val userCredentialsDao: UserCredentialsDao
}
